pkgs/games/maptool/default.nix

```diff
@@ -37,13 +32,15 @@ let
 
   javafxModules = [ "base" "controls" "media" "swing" "web" "fxml" "graphics" ];
 
+  appClasspath = "share/${pname}";
```

Comment:
We don't need to put this onto a variable

Reply:
Any time a path is used more than once in a package, I think it should be a variable. The name of the variable documents the purpose of the path and gives editors an unambiguous way to find all the places this directory is referenced. This variable is used once when constructing the `classpath` variable, and once when copying the JARs to $out.